packages/@aws-cdk/integ-tests-alpha/test/assertions/providers/lambda-handler/http.test.ts

```diff
@@ -113,7 +113,7 @@ describe('HttpHandler', () => {
     const response = await processEvent({ parameters: { url: 'x' } });
 
     // THEN
-    expect(response.apiCallResponse.body).toEqual({ key: 'value' });
```

Comment:
This change should not have any impact given `apiCallResponse` should only be used internally, but it is caused by the type change of the `HttpHandler` response from `HttpResponseWrapper` to `HttpResponseWrapper | { [key: string]: unknown }>`